app/helpers/application_helper.rb

```diff
@@ -34,4 +34,24 @@ def decorated_session
       @_decorated_session ||= SessionDecorator.new
     end
   end
+
+  def sp_session
+    session[:sp]
+  end
+
+  def loa1_signup_context?
+    sp_session && current_user && !current_user.recovery_code.present?
+  end
+
+  def loa3_context?
+    sp_session && sp_session[:loa3] && current_user.recovery_code.present?
```

Comment:
I must have misunderstood the comment https://github.com/18F/identity-private/issues/1333#issuecomment-277795571. It seemed to me that if the user never finished setting up their account, i.e. viewing their recovery code, that their account was supposed to be deleted.